frontend/src/components/CustomTimePicker/CustomTimePicker.styles.scss

```diff
@@ -119,3 +119,42 @@
 		color: var(--bg-slate-400) !important;
 	}
 }
+
+.date-time-popover-footer {
```

Comment:
make sure this name doesn't conflict with any generic classname, (not sure why but this seemed like some library might be having these kind 😅)

Reply:
Done :+1: